docker-compose.smoke-tests.yml

```diff
@@ -11,6 +11,7 @@ services:
         - https_proxy
         - no_proxy=remote-webdriver
         - IDAM_URL
+        - OAUTH2_CLIENT_SECRET
```

Comment:
The same has to be done in `cmc-integration-tests` repo

Reply:
done
https://github.com/hmcts/cmc-integration-tests/pull/176